### DISCUSSION TITLE
Easy way to remove a custom attribute from hundreds of entries

### DISCUSSION THREAD

**Discussion by JohnLGalt** [keepassxreboot/keepassxc#9737] [Q&A]
Hey, all, 

I suspect the answer is "manually remove each one", but I'm asking in case there is an easier way.

Long time KP2 user, switched to KPXC a couple years ago or so, and simply imported my extensive database from KP2 to KPXC.

I used an extension to KP2 called 'Have I Been Pwned', and those ghost attributes are still associated with nearly every entry in my database.

I already know how to search for that attribute and get a list of all entries that contain it. But is there a way to easily delete them all in a single step, or am I stuck going through each entry one by one?

I also search both issues and discussions for 'delete attribute', so if the answer has already been stated, I apologize, I missed it.

**Comment by droidmonkey** [comment]
I believe you could do this with the CLI tool. I am not sure how, but you can review the man page. Otherwise, you would have to export as XML, eliminate the attribute with a regex in a text editor, and then reimport. Caution, this would cause loss of attachments.

**Reply by JohnLGalt** [reply]
Awesome, I'll check it out and if I can find a way I'll come back and mark this as the answer.

And - apologies. I never even considered the CLI tool lol.

If I went the XML route (which I'm much more familiar with when it comes to editing via regEx in NPP), if I were to perform a search for all entries without attachments in the main GUI app, then export, it would definitely only export what matched my search parameters, right?

**Reply by droidmonkey** [reply]
We haven't implemented export selected entries yet

**Reply by JohnLGalt** [reply]
OK, then that nixes that idea lol - I have entirely too many entries with attachments. Doing those manually all over again is probably just as time consuming as just removing that attribute from all entries by hand.
